index.html

```diff
@@ -222,40 +222,48 @@ <h3 id="rec-track"> Normative Specifications </h3>
             <p class="draft-status"> <b>Draft state:</b> <a href="http://www.w3.org/TR/2016/CR-presentation-api-20160714/">
                 CR</a> / Stable </p>
             <p class="milestone">
-              <b>Expected completion:</b> Q4 2019 &mdash; This Working Group
-              does not anticipate further changes to this specification. The
-              Working Group is awaiting satisfaction of
-              the <a href="#success-criteria">Success Criteria</a>, which
-              require two interoperable implementations of the Presentation API.
+              <b>Expected completion:</b> Q4 2019 &mdash; The Working Group is
+              awaiting satisfaction of the <a href="#success-criteria">Success
+              Criteria</a>, which require two interoperable implementations of
+              the Presentation API.  The Working Group will also await
+              demonstration of interoperability between browsers and devices
```

Comment:
It might be clearer to say "between *multiple* browsers and *multiple* devices".  In particular, the point I'd like to be clear is that the interoperability criterion for advancing these specifications should require multiple browser implementations to interoperate with multiple device implementations.  That is, that it's not sufficient to just have a web page interoperate with multiple browser+device pairs that are using proprietary communications between them.  I think that's the most important point in the [comments I submitted on the charter](https://lists.w3.org/Archives/Public/public-new-work/2018Jan/0003.html).

Reply:
Done.